### ISSUE TITLE
Plugin is reported as not thead-safe when running with -T

### ISSUE THREAD

**Issue by dimas** [tcurdt/jdeb#194]
When you use Maven's -T option to run build in parallel, it complains that:

```
[WARNING] *****************************************************************
[WARNING] * Your build is requesting parallel execution, but project      *
[WARNING] * contains the following plugin(s) that are not marked as       *
[WARNING] * @threadSafe to support parallel building.                     *
[WARNING] * While this /may/ work fine, please look for plugin updates    *
[WARNING] * and/or request plugins be made thread-safe.                   *
[WARNING] * If reporting an issue, report it against the plugin in        *
[WARNING] * question, not against maven-core                              *
[WARNING] *****************************************************************
[WARNING] The following plugins are not marked @threadSafe in Project Test
[WARNING] org.vafer:jdeb:1.3
[WARNING] *****************************************************************
```

**Comment by ebourg** [comment]
Duplicate of  #123
